### PR TITLE
please accept my commit. All test work.

### PR DESCRIPTION
**Pull request by BossNeo**
tommorow will continue with more fixture to be more native. I eddit all no native meanings and merge some words , but sure need more .

### CODE REVIEW

**Comment by hristo-atanasov** [comment]
I'll want to put on hold this PR until I preview it and test it as me and other translators were having difficult times explaining bulgarian grammar and spelling to @BossNeo.

**Review by hristo-atanasov — CHANGES_REQUESTED** [review]
In `languages.yaml` bulgarian language is written wrong. In bulgarian language only country name, but not the language should start with capital letter. And that is a must. So when written in bulgarian the right is "българки" or "българки език" (bulgarian language) and not "Български". The rules allows the first letter of "български" only if it was used in sentence and if t is the first word (e.g. the beginning) of the sentence. So `languages.yaml` edit is wrong.

In `_common.yaml` you have used `зона` (in english `zone`) translating tottally different word - `област` (in english `area`). Zones are used for locations, so this word should not be used in other context. Even in bulgarian these two terms are different. So it is a big no. Consider using `област` for `area` as in Home Assistant Frontend to reduce user confusion.  Another mistake is replacing lines 43 and 45, as using "по фаренхайд" and "по целзий" is the oficial one in Bulgaria. The voice assistant should match bulgarian sentences not only the unit of measurment. Yes, the units of measurment are "целзий" and "фаренхайт", but when they are used in sentence they should be prepended with `по`.  It is like `by` or `using` in english. Because the main unit of measurment is `degree[s]` and it becomes "25 degrees `by` celsius" (in bulgarian "25 градуса `по` целзий") and it is reqired in our language. So in case of "Set kitchen temperature to 25C" we have to translate it as "Настрой температурата в кухнята на 25 градуса **`по`** целзий". In other words your edit of this file is useless.

In `_fixtures.yaml` you have used `()` instead of `[]` in almost all cases. So as an example on line 3 you wrote "кухня(та)" which returns one result "кухнята" instead of two results - "кухня" and "кухнята". As you have used it wrong not only in one document, this invalidates your entire PR. Test may work, but it is not even close to the right.

**Comment by hristo-atanasov** [comment]
I prefer to leave this here until other bulgarian colaborator[s] say something or someone else reject this PR.

**Review by hristo-atanasov — CHANGES_REQUESTED** [review]
In the second commit I see the same mistakes like in the first one - using () instead of [] like in `_common.yaml` line 52 and `_fixtures.yaml` lines 3, 11 and 23. Also I see bulgarian grammar mistakes on line 45 in the part `[по|в] фаренхайт` - when the next word starts with letter `в` (english `v`) or `ф` (english `f`), like in `фаренхайт`, the rule is to prepend `във` instead of `в`. Also in this case the right word is `по`, you can't say `в`. These are basic rules in bulgarian language. `Please, consider refreshing Your bulgarian grammar before messing with translations!` Writing the reviews and pointing Your mistakes takes twice as long as it would take to translate this files or make the changes myself. For those 2 hours explaining You simple bulgarian gramatics in facebook and discord I could do a lot of useful work.

File: sentences/bg/_common.yaml [hunk 1]
```diff
@@ -40,20 +40,21 @@ lists:
...
+  brightness: "{brightness} [процент(а)]"
+  what_is: "(какво е| каква е | какви са | какъв е | колко е | колко са)"
+  temp: "(температур(а|ата) | градус(а)"
   temperature: "{temperature} [градуса] [{temperature_unit}]"
   turn_on: (пусни | включи | светни)
-  turn_off: (спри | изключи | изгаси)
-  set: (направи | сложи | постави | смени | промени | настрой | задай | нагласи)
-  close: (затвори | смъкни | спусни | закрий | пусни)
+  turn_off: (спри | изключи | изгаси | угаси )
+  set: (направи | сложи | постави | смени | промени | настрой | задай | нагласи | тури)
```
Comment (dobromir-hristov) [inline sentences/bg/_common.yaml:56]: Тури тая лампа червена :D

Comment (BossNeo) [reply sentences/bg/_common.yaml:56]: > Тури тая лампа червена :D

Именно , българският език е богат не виждам защо да не се позволява.  Ако има проблем с разчитането на всички думи разбира се ще трябва първа да отпадне. Същото е и с вече предложените и одобрени кепенци .

Comment (dobromir-hristov) [reply sentences/bg/_common.yaml:56]: Мда кепенците бих казал че са излишни, колегата се е залиснал.

Вече има пълни преводи на файловете, можем лесно малко по малко да изчистваме нещата. По-добре малки PRs да се коментират нещата по-лесно.

Comment (BossNeo) [reply sentences/bg/_common.yaml:56]: Аз бях тръгнал с коментари , но ми писаха че коментарите трудно се следят и трудно се приемат, и да  съм правил PR в последния си edit съм оправил всички забележки които имаше. Като гледам не сме много хора  може да  направим някой общ чат и да се коментират  нещата.

Comment (hristo-atanasov) [reply sentences/bg/_common.yaml:56]: `Кепенците` и `тури` ще бъдат нашата малка тайна за забава на тези, които четат кода. На къде без тези два бисера. Че даже могат да се използват и в едно изречение: "Компютър, тури кепенците надоле!" :D :P

**Pull request closed without merging**